applications/asset_tracker_v2/overlay-lwm2m.conf

```diff
@@ -6,14 +6,19 @@ CONFIG_LWM2M_RW_SENML_JSON_SUPPORT=y
 CONFIG_LWM2M_RW_OMA_TLV_SUPPORT=y
 CONFIG_LWM2M_VERSION_1_1=y
 CONFIG_LWM2M_ENGINE_STACK_SIZE=3072
-CONFIG_LWM2M_COAP_MAX_MSG_SIZE=2304
```

Comment:
Now You use 512 only?. Set 1280.

Reply:
It defaults to the CoAP block size which is set to `1024`.

Reply:
Any particular reason why it should default to `1280`?

Reply:
OK if everything is working then it is fine.